### ISSUE TITLE
urllib3.exceptions.ResponseNotChunked from watching namespaced configmap or secret 

### ISSUE THREAD

**Issue by JTarball** [kubernetes-client/python#1047]
**What happened (please include outputs or screenshots)**:
The following api calls against CoreV1Api
list_namespaced_secret 
list_namespaced_config_map 

fail straight away with w.stream() if the resource version is added 

`
    raise ResponseNotChunked(
urllib3.exceptions.ResponseNotChunked: Response is not chunked. Header 'transfer-encoding: chunked' is missing.
`
**What you expected to happen**:
Not fail straight away the below code works for list_namespaced_event etc

**How to reproduce it (as minimally and precisely as possible)**:
`
        w = watch.Watch()

        for event in w.stream(self.v1api.list_namespaced_config_map, namespace, resource_version=resource_version):
`

**Anything else we need to know?**:
resource version is definitely set to a string 
- not adding the resource version kwarg makes the code work fine

**Environment**:
- Kubernetes version (`kubectl version`): Major:"1", Minor:"14", GitVersion:"v1.14.7",
- OS (e.g., MacOS 10.13.6): 10.15.1
- Python version (`python --version`): Python 3.8.1 
- Python client version (`pip list | grep kubernetes`): 11.0.0b2

**Comment by roycaihw** [comment]
Could you share more code about how you extracted the resource version? Also could you enable the [debug](https://github.com/kubernetes-client/python/blob/master/kubernetes/client/configuration.py#L76) switch and share the API response?

**Comment by kramerul** [comment]
We are running in the same issue.

```
    for e in w.stream(self.custom.list_namespaced_custom_object, "core.gardener.cloud", "v1beta1", self.namespace, "shoots", label_selector="name=" + name):
  File "/root/.local/lib/python3.6/site-packages/kubernetes/watch/watch.py", line 144, in stream
    for line in iter_resp_lines(resp):
  File "/root/.local/lib/python3.6/site-packages/kubernetes/watch/watch.py", line 46, in iter_resp_lines
    for seg in resp.read_chunked(decode_content=False):
  File "/root/.local/lib/python3.6/site-packages/urllib3/response.py", line 743, in read_chunked
    "Response is not chunked. "
urllib3.exceptions.ResponseNotChunked: Response is not chunked. Header 'transfer-encoding: chunked' is missing.

```

**Comment by schneesu** [comment]
According to changelog, this bug fixed in v17.17.0
- https://github.com/kubernetes-client/python/blob/master/CHANGELOG.md#v17170
- https://github.com/kubernetes-client/python-base/pull/231

thanks to dhague
